src/components/SykmeldingViews/SykmeldingView/SykmeldingContainer.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { SykmeldingFragment } from '../../../fetching/graphql.generated';
+
+import SykmeldingArbeidsgiverContainer from './SykmeldingArbeidsgiverContainer';
+import SykmeldingSykmeldtContainer from './SykmeldingSykmeldtContainer';
+
+interface Props {
+    sykmelding: SykmeldingFragment;
+    expandable?: boolean;
+    arbeidsgiver?: boolean;
+}
+
+const SykmeldingContainer: React.FC<Props> = ({ sykmelding, expandable = false, arbeidsgiver }: Props) => {
+    return (
+        <>
+            {arbeidsgiver ? (
+                <SykmeldingArbeidsgiverContainer sykmelding={sykmelding} expandable={expandable} />
+            ) : (
+                <SykmeldingSykmeldtContainer sykmelding={sykmelding} />
+            )}
+        </>
+    );
+};
```

Comment:
kan vi ikke egentlig bare droppe hele denne kontaineren og bruke de to underkomponentene direkte?